network-api/networkapi/templates/pages/base.html

```diff
@@ -31,7 +31,7 @@
     {% block stylesheets %}
       <link rel="stylesheet" href="{% static "_css/main.compiled.css" %}">
       <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i,800,900">
-      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i">
+      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,500,600,700,300i">
```

Comment:
if this is something new needed for design, maybe they want a change to say one of the other font weights is fine

Reply:
Good question. I asked the design team in Slack.

Reply:
We're not using 500. Ill have that removed shortly.